package.json

```diff
@@ -4,55 +4,30 @@
   "description": "Simorgh",
   "main": "./src/client.js",
   "scripts": {
-    "amp:validate": "wait-on http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost.bbc.com:7080/igbo.amp",
-    "audit:ci": "audit-ci --low",
+    "amphtml-validator:validate": "wait-on http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost.bbc.com:7080/igbo.amp",
     "bbcA11y": "wait-on -t 20000 http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po && bbc-a11y",
-    "build:ci": "export NODE_ENV=production && rm -rf build && npm run build:test && npm run build:live",
-    "build": "rm -rf build && cp envConfig/local.env .env && npm run webpack:prod && npm run test:size",
+    "build": "rm -rf build && cp envConfig/local.env .env && NODE_ENV=production webpack && ./scripts/bundleSize.sh",
+    "build:live": "cp envConfig/live.env .env && NODE_ENV=production webpack",
     "build:storybook": "build-storybook -c .storybook -o storybook_dist",
-    "build:test": "cp envConfig/test.env .env && npm run webpack:prod",
-    "build:live": "cp envConfig/live.env .env && npm run webpack:prod",
-    "cypress:proxy": "export NO_PROXY=localhost,localhost.bbc.com && cypress",
-    "cypress": "npm run cypress:proxy -- run",
-    "cypress:interactive": "npm run cypress:proxy -- open",
-    "cypress:3rdParty": "npm run cypress -- --project ./3rdPartyCypress/.",
+    "build:test": "cp envConfig/test.env .env && NODE_ENV=production webpack",
```

Comment:
(minor) the `NODE_ENV=production` seems duplicated between these npm scripts and the Makefile...